go.mod

```diff
@@ -7,31 +7,28 @@ require (
 	github.com/briandowns/spinner v1.6.1
 	github.com/dapr/dapr v1.6.0
 	github.com/dapr/go-sdk v1.0.0
-	github.com/docker/docker v17.12.0-ce-rc1.0.20200916142827-bd33bbf0497b+incompatible
-	github.com/emicklei/go-restful v2.10.0+incompatible // indirect
-	github.com/fatih/color v1.10.0
-	github.com/garyburd/redigo v1.6.0 // indirect
+	github.com/docker/docker v20.10.12+incompatible
+	github.com/fatih/color v1.13.0
 	github.com/gocarina/gocsv v0.0.0-20190426105157-2fc85fcf0c07
 	github.com/hashicorp/go-retryablehttp v0.5.3
 	github.com/hashicorp/go-version v1.3.0
-	github.com/mattn/go-runewidth v0.0.8 // indirect
 	github.com/mitchellh/go-ps v0.0.0-20190716172923-621e5597135b
 	github.com/nightlyone/lockfile v0.0.0-20180618180623-0ad87eef1443
-	github.com/olekukonko/tablewriter v0.0.2
+	github.com/olekukonko/tablewriter v0.0.5
 	github.com/phayes/freeport v0.0.0-20180830031419-95f893ade6f2
 	github.com/pkg/browser v0.0.0-20180916011732-0a3d74bf9ce4
 	github.com/shirou/gopsutil v3.21.4+incompatible
-	github.com/spf13/cobra v1.1.3
-	github.com/spf13/viper v1.7.0
+	github.com/spf13/cobra v1.3.0
+	github.com/spf13/viper v1.10.0
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/sys v0.0.0-20211007075335-d3039528d8ac
+	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e
 	gopkg.in/yaml.v2 v2.4.0
-	helm.sh/helm/v3 v3.5.3
-	k8s.io/api v0.20.2
-	k8s.io/apiextensions-apiserver v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/cli-runtime v0.20.2
-	k8s.io/client-go v0.20.2
+	helm.sh/helm/v3 v3.8.0-rc.1.0.20220215042704-bc2f1e08a43a
```

Comment:
@shubham1172 Is your commit the only change from the last release (3.8.0)? If not we have changes in the client which are not yet released right?

Reply:
@mukundansundar, I don't think so. This points to my commit hash, which has a delta of around 27 commits after v3.8.0. 

>If not we have changes in the client which are not yet released right?

Can you please clarify on this part, I did not get it.

Reply:
if there are more commits after the 3.8 release than the single commit that you made, we have unreleased code also as part of the client right?

Reply:
That's true. Their next release is 3.8.1 on March 09, so we can choose to wait as well.

Reply:
Let's wait.